bees/emailbee/emailbee.go

```diff
@@ -55,9 +55,12 @@ func (mod *EmailBee) Action(action bees.Action) []bees.Placeholder {
 		m.SetHeader("From", mod.username)
 		m.SetHeader("To", to)
 		m.SetHeader("Subject", subject)
-		if plainText != "" {
-			m.SetBody("text/plain", plainText)
-		}
+
+		// remove the HTML tags, such as <html></html> and so on
+		re, _ := regexp.Compile("\\<[\\S\\s]+?\\>")
```

Comment:
Why would that be needed? Whatever the user passes into "plaintext", we should use it and don't change the contents.